README.md

```diff
@@ -64,7 +101,18 @@ argument. Finally, demonstrate calling `doSomething` with a function.**
 
 Your Answer:
 ```js
-// write code here
+The language is quite confusing; I've a hard time understanding the main concept. As for my understanding goes, argument(s) for a function goes inside (); and then included inside the function like we've been doing with examples like: function doSomething( num1, num2) {
+num1*num2;
+}
+
+//But in this particular question, I'd assume it goes as follows
+
+function doSomething (thingToDo) {
+thingToDo;
```

Comment:
The only thing you're missing is parenthesis here, which call the function passed in:

```js
thingToDo();
```
Happy to talk more about this in person